lib/node/Status.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const clone = require('../util/clone');
+
```

Comment:
I generally grouped the ones that start with a non-capital letter first and the ones starting with a capital letter second, but I don't have a strong opinion on this, what do you think?